website/docs/d/db2_instance.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "Db2 SaaS"
```

Comment:
I see different category here in allowed-subcategories its referred as "Db2 Database" . These two should be same

Reply:
Done , it should be "Db2 SaaS"